### ISSUE TITLE
Adding new variable for electricity merit order prices

### ISSUE THREAD

**Issue by IngeborgGraabak** [openENTRANCE/nomenclature#19]
In the present version of the common data format, there are two variables for electricity prices:
i)	"Price|Secondary Energy|Electricity" with the explanation "electricity price at the secondary level, i.e. for large scale consumers (e.g. aluminum production). Prices should include the effect of carbon prices."
ii)	"Price|Final Energy|Residential|Electricity" with the explanation "Electricity price at the final level in the residential sector. Prices should include the effect of carbon prices"

FANSI provides electricity prices per hour and regions according to merit-order. The merit order prices are independent of type of consumption, e.g. "Residental", "Large consumers" etc.
I suggest adding a new variable "Price|Final Energy|Merit order|Electricity" with the explanation "Electricity price at the final level calculated according to the merit order principle"

**Comment by danielhuppmann** [comment]
Thanks @IngeborgGraabak for raising this question.

If I understand you correctly, you want to report the marginal cost of supplying electricity (ie the point on the merit order cost curve where generation and demand are equal). This should (in principle) be identical to the short-term spot market price.

From the point of the energy supply chain, this would be at the secondary energy level - if it's not related to consumption, it shouldn't be final energy. So I would rather
 - adapt the definition of `Price|Secondary Energy|Electricity` to be related to the market price
 - add a new variable `Price|Final Energy|Industry|Electricity` to specifically report the price at the consumption level for large-scale consumers
 - if you need to report the spot market price and the marginal cost separately, then add another variable `Price|Secondary Energy|Electricity|Marginal Cost` and explain how the the two concepts are distinct.

**Comment by arght** [comment]
I understand that there are two points of view:

- One is the short-run marginal cost (SRMC), which is the output of a centralized system-wide model. It can be obtained as a dual variable or as a result of a postprocessing of the loading order. This is commonly used as a proxy of the spot market price. Only in market equilibirum models the market price is an explicit variable. For this ouput I suggest to use
Marginal Cost|Secondary Energy|Electricity
or
Market Price|Secondary Energy|Electricity
depending on the case

- The other is the market price as an input parameter for a consumer model, where industrial, comercial and households may have a different market price. For this I suggest to use:
Price|Final Energy|Industry|Electricity
Price|Final Energy|Comercial|Electricity
Price|Final Energy|Residential|Electricity

**Comment by danielhuppmann** [comment]
Thanks @arght - agree to add commercial and industrial electricity price variables to the already existing one for residential customers (see [here](https://github.com/openENTRANCE/nomenclature/blob/af58b3653e6af61bf48401467765fce916442916/variable/economy/economy.yaml#L106)) - would be nice if you could start a pull request for that?

Regarding the distinction between SRMC and spot market price, I would still like to wait for a response by @IngeborgGraabak whether the two concepts are really different - and if not, we use the existing variable (see [here](https://github.com/openENTRANCE/nomenclature/blob/af58b3653e6af61bf48401467765fce916442916/variable/economy/economy.yaml#L155)).

**Comment by IngeborgGraabak** [comment]
From my point of view, we can use the same variable for the SRMC and the spot market price. It is fine if we adapt the definition of Price|Secondary Energy|Electricity to be related to the market price.

**Comment by sandrinecharousset** [comment]
Hi everyone,
My point of view is that there exist different categories of ‘prices’ that are different:
- first one is what we call marginal cost. In a perfect and rational market it should be equal to the ‘market price ‘ but in reality it is not. This is what usual physical models (like plan4eu and maybe fansi compute (in reality an approximation as it has been proved that the real marginal cost in general does not exist...). It usually is the best approximation (or forecast) of market price that can be done with a physical model
- second is market prices: here you have many different, spot price and different forward prices (related to different time horizons ); spot and forward prices come only from pure energy market but now are emerging other kinds of markets like capacity markets, system services markets... 
- third is the prices paid by consumers. Some very big consumers are allowed to buy directly in markets but they are very few (and must ensure many conditions like keeping an equilibrium between their forecast of consumption and sourcing contracts...); those ‘prices’ are not equal to market prices but are generally linked to them. Except in the case of regulated prices.
I thus would be in favour of identifying those differences between marginal costs, market prices and retail prices
Wishing you happy easter

**Comment by IngeborgGraabak** [comment]
I agree that we have three categories. FANSI calculates the marginal prices as you suggest. For me it is OK if we split into these three categories.
